src/App.js

```diff
@@ -1,5 +1,60 @@
+const { Console } = require("@woowacourse/mission-utils");
+const BaseballGame = require("./BaseballGame");
+const { COMMAND, COUNT } = require("./constants/game");
+
+const InputView = require("./view/InputView");
+const OutputView = require("./view/OutputView");
+
 class App {
-  play() {}
+  constructor() {
+    this.baseballGame = new BaseballGame();
+  }
+
+  play() {
+    OutputView.printStart();
+    this.start();
+  }
+
+  start() {
+    this.baseballGame.createAnswerNumbers();
+
+    this.readPlayerInput();
+  }
+
+  readPlayerInput() {
+    InputView.readPlayerInput(this.compareAnswer.bind(this));
+  }
+
+  compareAnswer(threeDigistsNumber) {
+    const count = this.baseballGame.getCount(threeDigistsNumber);
+    OutputView.printCountResult(count);
```

Comment:
`count`보다는 `countResult`라는 네이밍이 좀 더 명시적이지 않나 생각이 듭니다. `this.baseballGame.getCount()`은 볼과 스트라이크 카운트의 결과를 리턴해주고 있고, `OutputView.printCountResult()`은 카운트의 결과를 출력하기 때문에 이름을 좀 더 추론할 수 있으면서 통일성 있게 만들면 어떨까 싶습니다👍

```suggestion
    const countResult = this.baseballGame.getCountResult(threeDigistsNumber);
    OutputView.printCountResult(countResult);
```

Reply:
변수명 짓기가 세상에서 제일 어려운 것 같습니다 ㅠㅠ

`countResult`가 제 코드의 흐름에 더 적절한 것 같네요 ㅎㅎ 좋은 의견 감사합니다✔